### PR TITLE
feat: .zulu command

### PR DESCRIPTION
**Pull request by Earthsam12**
## Description

adds command .zulu to show the current time in a given UTC-offset timezone. this can be useful given the fact that a majority of active users have their UTC timezone in their nickname (e.g. [Z-4], [Z+11])

## Test Results

image

## Discord Username

Earthsam12#5545

### CODE REVIEW

**Comment by benw202** [comment]
This will probably need some error handling in it.

**Comment by Earthsam12** [comment]
> This will probably need some error handling in it.

I added a simple line to account for when `parseFloat` doesn't 'filter out' bad inputs. I couldn't find any other cases that should be handled differently, except for maybe not allowing more a than ±12 (13?) timezone

**Review by benw202 — APPROVED** [review]

**Comment by benw202** [comment]
LGTM!

File: src/commands/index.ts [hunk 1]
```diff
@@ -223,9 +224,11 @@ const commands: CommandDefinition[] = [
     abbreviations,
     airac,
     mico,
+    translate,
```
Comment (benw202) [inline src/commands/index.ts:227]: Repeated line?

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/commands/index.ts (hunk 1 above)
Comment (benw202) [inline src/commands/index.ts:227]:
```suggestion
```

**Review by benw202 — CHANGES_REQUESTED** [review]

**Review by benw202 — APPROVED** [review]

**Pull request merged**